dodola/tests/test_services.py

```diff
@@ -197,27 +206,22 @@ def test_regrid_methods(regrid_method, expected_shape):
         out="an/output/path.zarr",
         method=regrid_method,
         storage=fakestorage,
+        domain_file="a/domainfile/path.zarr",
     )
     actual_shape = fakestorage.read("an/output/path.zarr")["fakevariable"].shape
     assert actual_shape == expected_shape
 
 
 @pytest.mark.parametrize(
-    "target_resolution, expected_shape",
+    "expected_shape",
     [
         pytest.param(
-            1.0,
             (180, 360),
-            id="Regrid to global 1.0° x 1.0° grid",
-        ),
-        pytest.param(
-            2.0,
-            (90, 180),
-            id="Regrid to global 2.0° x 2.0° grid",
+            id="Regrid to domain file grid",
         ),
     ],
 )
-def test_regrid_resolution(target_resolution, expected_shape):
+def test_regrid_resolution(expected_shape):
     """Smoke test that services.regrid outputs with different regrid methods
 
     The expected shape is the same, but change in methods should not error.
```

Comment:
I think I botched this docstr. It should say

```
"""Smoke test that services.regrid outputs with different grid resolutions
"""
```

Would you mind correcting this, if we keep this test?

Reply:
Thank you! This is resolved.